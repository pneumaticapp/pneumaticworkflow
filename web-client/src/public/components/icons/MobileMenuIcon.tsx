/* eslint-disable */
/* prettier-ignore */
import * as React from 'react';

export const MobileMenuIcon = () => (
  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 26 17">
    <rect x="0.5" y="0.5" width="25" height="1" />
    <rect x="0.5" y="7.5" width="25" height="1" />
    <rect x="0.5" y="15.5" width="25" height="1" />
  </svg>
);
