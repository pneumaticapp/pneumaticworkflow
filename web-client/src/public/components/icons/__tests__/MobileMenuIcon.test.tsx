/* eslint-disable */
/* prettier-ignore */
import * as React from 'react';
import { shallow } from 'enzyme';

import { MobileMenuIcon } from '../MobileMenuIcon';

describe('Mobile Menu Icon', () => {
  it('корректно рендерится', () => {
    const wrapper = shallow(<MobileMenuIcon />);

    expect(wrapper).toMatchSnapshot();
  });
});
