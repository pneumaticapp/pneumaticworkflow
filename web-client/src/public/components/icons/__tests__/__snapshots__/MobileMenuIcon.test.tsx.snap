// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mobile Menu Icon корректно рендерится 1`] = `
<svg
  viewBox="0 0 26 17"
  xmlns="http://www.w3.org/2000/svg"
>
  <rect
    height="1"
    width="25"
    x="0.5"
    y="0.5"
  />
  <rect
    height="1"
    width="25"
    x="0.5"
    y="7.5"
  />
  <rect
    height="1"
    width="25"
    x="0.5"
    y="15.5"
  />
</svg>
`;
